cmd/litcli/autopilot.go

```diff
@@ -39,10 +40,48 @@ var addAutopilotSessionCmd = cli.Command{
 	Name:      "add",
 	ShortName: "a",
 	Usage:     "Initialize an Autopilot session.",
-	Description: "Initialize an Autopilot session.\n\n" +
-		"   If set for any feature, configuration flags need to be " +
-		"repeated for each feature that is registered, corresponding " +
-		"to the order of features.",
+	Description: `
+	Initialize an Autopilot session.
+
+	If one of the 'feature-' flags is set for any 'feature', then that flag
+	must be provided for each 'feature'.
+
+	The rules and configuration options available for each feature can be
+	seen in the 'autopilot features' output.
+	
+	An example call reads:
+	
+	./litcli autopilot add --label=customRules \
+	--feature=SampleFeature \
+	--feature-rules='{
+		"rules": {
+			"rate-limit": {
+				"rate_limit": {
+					"read_limit": {
+						"iterations": 10,
+						"num_hours": 1
+					},
+					"write_limit": {
+						"iterations": 1,
+						"num_hours": 1
+					}
+				}
+			},
+			"channel-restriction": {
+				"channel_restrict": {
+					"channel_ids": [
+						18283782789, 
+						22891928322, 
+						31878293727
+					]
+				}
+			}
```

Comment:
nit:
Since this enables users to set any of the rules now, I do think that:

1.  It makes sense to give an example for all types rules the user can set, i.e. also include examples of the `channel-policy-bounds`, `history-limit` & `peer-restriction` rule types here.
2. Clarify in the docs that if a specific rule type is not set, the default values will be used.

Reply:
Good calls, concerning 1, I have added the most interesting rules for `AutoFees`, but left others away since I think it would add too much help output and it should be clear how to apply the pattern. I updated the help text a bit more to give more hints on how to correctly set rules. I've added that comment for 2.

Reply:
> but left others away since I think it would add too much help output and it should be clear how to apply the pattern

Ok thanks for the edits :)! Sure, I see your point, let's avoid adding too much output. In case we end up getting lot's of user questions of how to apply a specific rule in the future, we can always revisit this and add more info :)!